sdk/monitor/opentelemetry-exporter-azuremonitor/src/main/java/com/azure/opentelemetry/exporter/azuremonitor/AzureMonitorExporter.java

```diff
@@ -0,0 +1,784 @@
+        dbSystems.add("postgresql");
+        dbSystems.add("sqlite");
+        dbSystems.add("other_sql");
+        dbSystems.add("hsqldb");
+        dbSystems.add("h2");
+
+        SQL_DB_SYSTEMS = Collections.unmodifiableSet(dbSystems);
+    }
+
+    private final MonitorExporterClient client;
+    private final ClientLogger logger = new ClientLogger(AzureMonitorExporter.class);
+    private final String instrumentationKey;
+    private final String telemetryItemNamePrefix;
+
+    /**
+     * Creates an instance of exporter that is configured with given exporter client that sends telemetry events to
+     * Application Insights resource identified by the instrumentation key.
+     *
+     * @param client The client used to send data to Azure Monitor.
+     * @param instrumentationKey The instrumentation key of Application Insights resource.
+     */
+    AzureMonitorExporter(MonitorExporterClient client, String instrumentationKey) {
+        this.client = client;
+        this.instrumentationKey = instrumentationKey;
+        this.telemetryItemNamePrefix = "Microsoft.ApplicationInsights." + instrumentationKey + ".";
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public CompletableResultCode export(Collection<SpanData> spans) {
+
+        try {
+            List<TelemetryItem> telemetryItems = new ArrayList<>();
+            for (SpanData span : spans) {
+                logger.verbose("exporting span: {}", span);
+                export(span, telemetryItems);
+            }
+            client.export(telemetryItems);
+            return CompletableResultCode.ofSuccess();
+        } catch (Throwable t) {
+            logger.error(t.getMessage(), t);
+            return CompletableResultCode.ofFailure();
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public CompletableResultCode flush() {
+        return CompletableResultCode.ofSuccess();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public CompletableResultCode shutdown() {
+        return CompletableResultCode.ofSuccess();
+    }
+
+    private void export(SpanData span, List<TelemetryItem> telemetryItems) {
+        Span.Kind kind = span.getKind();
+        String instrumentationName = span.getInstrumentationLibraryInfo().getName();
+        Matcher matcher = COMPONENT_PATTERN.matcher(instrumentationName);
+        String stdComponent = matcher.matches() ? matcher.group(1) : null;
+        if ("jms".equals(stdComponent) && !span.getParentSpanId().isValid() && kind == Span.Kind.CLIENT) {
+            // no need to capture these, at least is consistent with prior behavior
+            // these tend to be frameworks pulling messages which are then pushed to consumers
+            // where we capture them
+            return;
+        }
+        if (kind == Span.Kind.INTERNAL) {
+            if (span.getName().equals("log.message")) {
+                exportLogSpan(span, telemetryItems);
```

Comment:
(not required for beta.1) we don't need log handling, this is all hack to transport logs over spans, and not official OpenTelemetry (which will support Logs separately in the future)